<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCentralWidget</class>
 <widget class="QWidget" name="CCentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCentralWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="10,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="CGLWidget" name="glwidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="toolbox" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>231</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>151</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="upButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="vrpmapeditorqt.qrc">
                  <normaloff>:/VRPMapEditorQt/Resources/up.png</normaloff>:/VRPMapEditorQt/Resources/up.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>12</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="leftButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="vrpmapeditorqt.qrc">
                  <normaloff>:/VRPMapEditorQt/Resources/left.png</normaloff>:/VRPMapEditorQt/Resources/left.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>24</width>
                  <height>12</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="rightButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="vrpmapeditorqt.qrc">
                  <normaloff>:/VRPMapEditorQt/Resources/right.png</normaloff>:/VRPMapEditorQt/Resources/right.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>24</width>
                  <height>12</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="downButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="vrpmapeditorqt.qrc">
                  <normaloff>:/VRPMapEditorQt/Resources/down.png</normaloff>:/VRPMapEditorQt/Resources/down.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>12</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>18</number>
            </property>
            <property name="value">
             <number>13</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="invertedAppearance">
             <bool>true</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>231</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>132</width>
           <height>76</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0">
           <widget class="QCheckBox" name="showGrid">
            <property name="text">
             <string>Show Grid</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="showConnections">
            <property name="text">
             <string>Show Connections</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QComboBox" name="mapType">
              <item>
               <property name="text">
                <string>roadmap</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>satellite</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>terrain</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hybrid</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Map type</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>149</x>
           <y>10</y>
           <width>85</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>15</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>7</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Transparency</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Line Width</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CGLWidget</class>
   <extends>QWidget</extends>
   <header>GLwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vrpmapeditorqt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>upButton</sender>
   <signal>clicked()</signal>
   <receiver>CCentralWidget</receiver>
   <slot>UpClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftButton</sender>
   <signal>clicked()</signal>
   <receiver>CCentralWidget</receiver>
   <slot>LeftClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downButton</sender>
   <signal>clicked()</signal>
   <receiver>CCentralWidget</receiver>
   <slot>DownClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightButton</sender>
   <signal>clicked()</signal>
   <receiver>CCentralWidget</receiver>
   <slot>RightClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>CCentralWidget</receiver>
   <slot>SliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showGrid</sender>
   <signal>clicked()</signal>
   <receiver>CCentralWidget</receiver>
   <slot>ShowGridClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>CCentralWidget</receiver>
   <slot>ComboBoxIndexChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showConnections</sender>
   <signal>clicked()</signal>
   <receiver>CCentralWidget</receiver>
   <slot>ShowConnectionsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CCentralWidget</receiver>
   <slot>SliderValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CCentralWidget</receiver>
   <slot>LineWidthChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UpClicked()</slot>
  <slot>DownClicked()</slot>
  <slot>LeftClicked()</slot>
  <slot>RightClicked()</slot>
  <slot>SliderReleased()</slot>
  <slot>ShowGridClicked()</slot>
  <slot>ComboBoxIndexChanged(QString)</slot>
  <slot>ShowConnectionsClicked()</slot>
  <slot>SliderValueChanged(int)</slot>
  <slot>LineWidthChanged(int)</slot>
 </slots>
</ui>
